src/algebra/lie/solvable.lean

```diff
@@ -95,7 +95,7 @@ derived_series_of_ideal_le (le_refl I) (zero_le k)
 lemma derived_series_of_ideal_mono {I J : lie_ideal R L} (h : I ≤ J) (k : ℕ) : D k I ≤ D k J :=
 derived_series_of_ideal_le h (le_refl k)
 
-lemma derived_series_of_ideal_antimono {k l : ℕ} (h : l ≤ k) : D k I ≤ D l I :=
+lemma derived_series_of_ideal_antitone {k l : ℕ} (h : l ≤ k) : D k I ≤ D l I :=
```

Comment:
While you're at it, could you please add `@[mono]` attr to `derived_series_of_ideal_le`?